Team6220_PowerPlay/src/main/java/org/firstinspires/ftc/team6220_PowerPlay/testclasses/ColorDetectPipeline.java

```diff
@@ -0,0 +1,112 @@
+package org.firstinspires.ftc.team6220_PowerPlay.testclasses;
+
+import org.opencv.core.Core;
+import org.opencv.core.Mat;
+import org.opencv.core.MatOfPoint;
+import org.opencv.core.Rect;
+import org.opencv.core.Scalar;
+import org.opencv.core.Size;
+import org.opencv.imgproc.Imgproc;
+import org.openftc.easyopencv.OpenCvPipeline;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ColorDetectPipeline extends OpenCvPipeline {
+    private Size blurSize = new Size(25, 25);
+    private double erodeSize = 220;
+
+    public boolean isRunning = false;
+    public Rect detectedRect = new Rect();
+    public boolean rectDetected = false;
+    public int counter = 0;
+
+    private Mat frame = new Mat();
+    private Mat matte = new Mat();
```

Comment:
Is Mat short for matte or matrix?

Reply:
matrix

Reply:
matte means not shiny so you should change this to be matrix

Reply:
File no longer exists